### DISCUSSION TITLE
Open link with a click

### DISCUSSION THREAD

**Discussion by JekRock** [zellij-org/zellij#1859] [Q&A] [answered]
It would be nice to be able to open links in the terminal by clicking on them with the mouse.

And thank you for a great terminal workspace. I like it. Very convenient, easy to work with, and has good defaults. With tmux, all these stuff are much more complicated.

**Comment by mtdowling** [comment, marked as answer]
FWIU, your terminal should be thing identifying links. Alacritty does this, for example. Then in Zellij, hold down shift to click them (that makes the mouse click pass through to the CLI).

**Reply by JekRock** [reply]
Holding shift does work! Didn't know about that. Thank you!
